### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by medsonmoombe**
## The following were implemented in this project
- No linter errors
- Use Formspree
- Use flexbox
- pair programming

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -174,56 +289,89 @@ <h4 class="about-l">About me</h4>
...
+
+  <section class="form-section">
+    <article>I'm always interested
+         in hearing about new projects,
+         so if you'd like to chat 
+         please get in touch.
+    </article>
+         <div class="form-input">
+            <form action="https://formspree.io/f/xnqwbvpv" method="post">
+               <label for="name" class="ful-name">Full name</label>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly remove the label and add it as a  placeholder in the input tag, so the placeholder disappears when the user inputs the details

File: index.html [hunk 2]
```diff
@@ -174,56 +289,89 @@ <h4 class="about-l">About me</h4>
...
+               <input id="name" type="text" class="full-name" maxlength="30"><br>
+               <label for="email" class="email-adres">Email address</label>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly remove the label and add it as a  placeholder in the input tag, so the placeholder disappears when the user inputs the details

File: index.html [hunk 3]
```diff
@@ -174,56 +289,89 @@ <h4 class="about-l">About me</h4>
...
+               <input id="email" type="email" name="email" class="email-input"><br>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly add the "required" attribute to activate the validation on submit for all input tags👍🏿

File: index.html [hunk 4]
```diff
@@ -174,56 +289,89 @@ <h4 class="about-l">About me</h4>
...
+               <textarea class="text-sec" name="textarea" rows="10" cols="30" placeholder="Enter text here" maxlength="500"></textarea>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly add the "required" attribute to activate the validation on submit 👍🏿

File: index.html (hunk 3 above)
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly add the "required" attribute to activate the validation on submit for all input tags👍🏿

File: index.html [hunk 5]
```diff
@@ -174,56 +289,89 @@ <h4 class="about-l">About me</h4>
...
+  <section class="form-section">
+    <article>I'm always interested
+         in hearing about new projects,
+         so if you'd like to chat 
+         please get in touch.
+    </article>
+         <div class="form-input">
+            <form action="https://formspree.io/f/xnqwbvpv" method="post">
+               <label for="name" class="ful-name">Full name</label>
+               <input id="name" type="text" class="full-name" maxlength="30"><br>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly add the "required" attribute to activate the validation on submit for all input tags👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @medsonmoombe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ GitHub flow was correctly used
✔️  Good design



## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medsonmoombe** [comment]
## The following changes were implemented
- Add required attribute to all input field
- Remove label tag and add placeholders

**Review by FrederickMih — APPROVED** [review]
# Status: Approved! :tada: :tada: :champagne: 

Hi @medsonmoombe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**